templates/_layout.html

```diff
@@ -58,6 +58,9 @@
           <li class="p-navigation__link" role="menuitem"><a href="https://landscape.canonical.com/static/doc/user-guide/">Docs</a></li>
           <li class="p-navigation__link" role="menuitem"><a href="https://landscape.canonical.com/static/doc/api/">API docs</a></li>
         </ul>
+        <ul class="p-navigation__links--right" role="menu">
+          <li class="p-navigation__link" role="menuitem"><a href="https://landscape.canonical.com/login/authenticate">Login</a></li>
```

Comment:
can we parameterise this? the root URL will be different based on dev/staging/production

Reply:
We do not have a mechanism to change variables based on staging/production. What is the benefit of links to staging when the site is on staging? What would we link to from local development?

Reply:
Being on staging, clicking login and logging-in to production is not exactly what one would expect.

We'll have to either get a mechanism to change variables based on staging/production or otherwise do something based on a parameter that landscape sends to the brochure site when it calls it.

Reply:
While we normally don't change application behaviour between staging & production for interaction with external services. However, in this case it certainly makes sense to keep people on the same domain. As in, presumably https://staging.landscape.canonical.com will proxy http://landscape-brochure.staging.internal, and so you would certainly expect to remain on https://staging.landscape.canonical.com when you click "log in". (I would have thought we'd use https://staging.landscape.canonical.com to check our changes before release).

But couldn't this be simply achieved by making the link relative - `/login/authenticate`? This would work fine on both https://staging.landscape.canonical.com and https://landscape.canonical.com. It would break in local development, but that might not be a totally bad thing, as it wouldn't do what you expect anyway - it's a bit surprising to be suddenly kicked over to https://landscape.canonical.com when logging in to a site on `localhost`. Or, if you wanted that behaviour, you could add a local redirect in the app for `/login/.*`, which would only be hit in local development.

Thoughts?